tools/wpt/commands.json

```diff
@@ -0,0 +1,2 @@
+{"run": {"path": "run.py", "script": "run", "parser": "create_parser", "help": "Run tests in a browser",
+         "virtualenv": true, "install": ["requests"], "requirements": ["../wptrunner/requirements.txt"]}}
```

Comment:
It's necessary to install the browser/webdriver/prefs/etc. so it's a requirement for `wpt run` but not for wptrunner itself.